Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\n/shelf/\n'/.idea/\n/workspace.xml\n# Editor-based HTTP Client requests\n/httpRequests/\n# Datasource local storage ignored files\n/dataSources/\n/dataSources.local.xml\n\n
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	
+++ b/.idea/.gitignore	
@@ -1,6 +1,6 @@
 # Default ignored files
 /shelf/
-'/.idea/
+/.idea/
 /workspace.xml
 # Editor-based HTTP Client requests
 /httpRequests/
